com.microsoft.mrtk.graphicstools.unity/Samples~/MaterialGallery/Animations/EmissiveRotation.controller

```diff
@@ -0,0 +1,72 @@
+%YAML 1.1
```

Comment:
Is this animation controller used anywhere? Doesn't look like it is? 
![image](https://user-images.githubusercontent.com/13305729/177883364-614b7d1d-02f7-40de-8a34-6aaa7a46183c.png)

Reply:
Leftover from previous attempt, deleted.

Reply:
Originally a leftover, but I left it in since I need the controller now or the animation, see below.